backend/spec/views/spree/admin/shared/navigation_footer_spec.rb

```diff
@@ -6,16 +6,39 @@
   let(:user) { FactoryBot.build_stubbed(:admin_user) }
   before do
     allow(view).to receive(:try_spree_current_user).and_return(user)
+    @ability = Object.new.extend(CanCan::Ability)
+    allow(@controller).to receive(:current_ability).and_return(@ability)
   end
 
   it "has a a login-nav section" do
     render
     expect(rendered).to have_selector("#login-nav")
   end
 
-  it "has a user-account-link" do
-    render
-    expect(rendered).to have_link(user.email, href: Spree::Core::Engine.routes.url_helpers.edit_admin_user_path(user))
+  context "authorized user" do
+    it "has a user-account-link that links to edit_admin_user_path" do
+      @ability.can :admin, user
```

Comment:
you can move this ability setting in a before that lives into the context `authorized user` since it will be applied to all its scenarios.